### ISSUE TITLE
Multiple files selection

### ISSUE THREAD

**Issue by LucaGallinari** [openstream/No-Flash-Image-Uploader#13]
Am i missing something, or is not possible to select multiple files from the "file selection window"? By the way i noticed that you can upload multiple files selected one by one.

**Comment by nickweisser** [comment]
Multiple files upload is not supported afaik, but feel free to add support for it ;)

**Comment by nickweisser** [comment]
@markvds has added support for multiple files upload in 94ae769 which I just merged. Please test it, if it works I will tag it as a new version.

**Comment by markvds** [comment]
Unfortunately that's not true. At least that's not what I intended to do :) 

The uploader was not working on the Edit Widget Instance page using `Aijko_WidgetImageChooser` because of missing javascript files on the calling page. The only thing I did was adding these files.

**Comment by hughconway** [comment]
Agreed - #94ae769 doesn't seem to allow multi-select but multi-upload does work if the files are selected one at a time before pressing 'Upload' as stated by @LucaGallinari

**Comment by nraajput** [comment]
Please tell me what to do for multiple select option for images.

**Comment by nickweisser** [comment]
Sorry, but that's not how it works.

**Comment by nraajput** [comment]
Can it be customised ?

**Comment by nickweisser** [comment]
Either you try to implement this yourself or you don't. We cannot give dev support here.
